README.md

```diff
@@ -913,9 +908,7 @@ if let woodchuck = woodchuck, canChuckWood(woodchuck) {
 // combined because we just return
 guard let thingOne = thingOne,
     let thingTwo = thingTwo,
-    let thingThree = thingThree else {
-    return
-}
+    let thingThree = thingThree else { return }
```

Comment:
I feel like this should be the same case (multi-line) and therefore we should not return in the same line. What do you think?

Reply:
Good eye man! Changed it back here [4e3e2c0](https://github.com/ecobee/swift-style-guide/pull/3/commits/4e3e2c0c96da27d7ecc0ce2885e14f1b1a3a2a53)